charts/rollups-validator-node/values.yaml

```diff
@@ -823,13 +823,16 @@ redis:
   # -- Use bitnami Redis&reg; pod.
   enabled: true
   # -- redis.endpoint: e.g.: redis://localhost:6379
-  # REQUIRED if you disabled redis
+  # REQUIRED if you disabled redis and your redis is standalone
   endpoint:
+  # -- redis.clusterEndpoint: e.g.: redis://localhost:6379
+  # REQUIRED if you disabled redis and your redis is cluster
+  clusterEndpoint:
```

Comment:
acctually, it should be `clusterEndpoints`, since cartesi-node accepts a lists of redis nodes separated by commas.

```
    --redis-cluster-endpoints <REDIS_CLUSTER_ENDPOINTS>...
          Address list of Redis cluster nodes, defined by a single string separated by commas. If present, it supersedes `redis_endpoint`. A single endpoint can be enough as the client will discover other nodes automatically [env: REDIS_CLUSTER_ENDPOINTS=]
  ```